examples/dft/run_time_dispatching/CMakeLists.txt

```
@@ -17,22 +17,34 @@
 # SPDX-License-Identifier: Apache-2.0
 #===============================================================================
 
-# NOTE: user needs to set env var SYCL_DEVICE_FILTER to use runtime example (no need to specify backend when building with CMake)
+# NOTE: user needs to set env var ONEAPI_DEVICE_SELECTOR to use runtime example (no need to specify backend when building with CMake)
 include(WarningsUtils)
 
 
 # Build object from all example sources
 set(DFT_RT_SOURCES "")
 # Set up for the right backend for run-time dispatching examples
 # If users build more than one backend (i.e. mklcpu and mklgpu, or mklcpu and CUDA), they may need to
-# overwrite SYCL_DEVICE_FILTER in their environment to run on the desired backend
+# overwrite ONEAPI_DEVICE_SELECTOR in their environment to run on the desired backend
 set(DEVICE_FILTERS "")
 if(ENABLE_MKLGPU_BACKEND OR ENABLE_CUFFT_BACKEND OR ENABLE_ROCFFT_BACKEND OR ENABLE_PORTFFT_BACKEND)
   list(APPEND DFT_RT_SOURCES "real_fwd_usm")
-  list(APPEND DEVICE_FILTERS "gpu")
 endif()
 
-message(STATUS "SYCL_DEVICE_FILTER will be set to the following value(s): [${DEVICE_FILTERS}] for run-time dispatching examples")
+if(ENABLE_MKLGPU_BACKEND)
+  list(APPEND DEVICE_FILTERS "level_zero:gpu")
+endif()
+if(ENABLE_PORTFFT_BACKEND)
+  list(APPEND_DEVICE_FILTERS "*:gpu")
```

Sorry about the incremental findings...
```suggestion
  list(APPEND DEVICE_FILTERS "*:gpu")
```

---

No, thank you for finding these, much appreciated. Committed in [9c1b328](https://github.com/oneapi-src/oneMKL/pull/484/commits/9c1b32870da9b46355b7cdbe2afe71ad2c441563), Git auto resolved this comment but assuming that's fine.